packages/app/components/ManageTabContent.tsx

```diff
@@ -8,328 +16,359 @@ import passportFactoryJson from "@cabindao/nft-passport-contracts/artifacts/cont
-                    
-                }}
-            >
-                <ModalInput
-                    label={"Address"}
-                    value={sAddr}
-                    onChange={(e) => setSAddr(e.target.value)}
-                />
-            </Modal>
-            <Modal
-                isOpen={bulkAddOpen}
-                setIsOpen={setBulkAddOpen}
-                title="Add addresses"
-                onConfirm={() => {
-                    const contract = new web3.eth.Contract(getAbiFromJson(passportJson));
-                    contract.options.address = selectedOption;
-                    return Promise.all(bulkAddrList.map(addr => resolveAddress(addr, web3)))
-                      .then(addrList => new Promise<void>((resolve, reject) =>
-                        contract.methods.addMinters(addrList)
-                          .send({
-                              from: address,
-                          })
-                          .on("receipt", (receipt: TransactionReceipt) => {
-                              setAllowedUsers([...bulkAddrList,...allowedUsers]);
-                              setBulkAddOpen(false);
-                              setBulkAddrList([]);
-                              resolve();
-                          })
-                          .on("error", reject)
-                    )).catch((e: Error) => setToastMessage(`ERROR: ${e.message}`));
-                    
-                }}
-            >
-                <div> {`All the addresses should be under column named "address". All other columns will be ignored.`} </div>
-                <ModalInputBox>
-                    <Label label={"Upload CSV"}>
-                        <input
-                        type={"file"}
-                        accept={".csv"}
-                        onChange={async (e) => {
-                            if (e.target.files) {
-                                const f: File = e.target.files[0];
-                                Papa.parse<Record<string,string>, File>(f, {
-                                    header: true, 
-                                    complete: function(results) {
-                                        const addrsInCsv = results.data.map((result) => result['address']);
-                                        const relevantAddr = addrsInCsv.filter((addr) => (addr && !allowedUsers.includes(addr)));
-                                        setBulkAddrList(relevantAddr);
-                                    },
-                                    error: function(e) {
-                                        setToastMessage(`ERROR: ${e.message}`);
-                                    }
-                                })
-                            }
-                        }}
-                        />
-                    </Label>
-                </ModalInputBox>
-            </Modal>
-            <h1>Manage Private Passports</h1>
```

Comment:
this is the only notable removal in this file (title now handled by `index.tsx`). the rest is prettier